### DISCUSSION TITLE
AutoComplete ComboBox, start with input empty? How to do that?

### DISCUSSION THREAD

**Discussion by phantodev** [tailwindlabs/headlessui#2621] [Help]
Guys, my question I believe is very simple, how do I make it so that instead of my AutoComplete ComboBox input already having the first item in the Array, I would like it to be empty, showing only my placeholder, written "Type something here..."

Thanks

**Comment by moon86** [comment]
Did you find a way to do it ? I would need it too for Vue.
It seems to work for React but not with Vue.
